features/util.py

```diff
@@ -231,19 +233,35 @@ def build_debs(series: str, cache_source: bool = False) -> List[str]:
 
     curr_dir = os.getcwd()
     os.chdir(SBUILD_DIR)
-    logging.info("--- Running sbuild")
+
+    sbuild_cmd = [
+        "sbuild",
+        "--no-run-lintian",
+        "--resolve-alternatives",
+        "--no-clean-source",
+        "--arch",
+        "amd64",
+        "-d",
+        series,
+        SOURCE_PR_UNTAR_DIR,
+    ]
+    if chroot is not None:
+        sbuild_cmd += ["--chroot", chroot]
+    else:
+        # use ua-series-arch chroot if present
+        ua_chroot = "ua-{}-amd64".format(series)
+        proc = subprocess.Popen(
+            ["schroot", "--list"],
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
+        )
+        out, _ = proc.communicate(timeout=1)
+        if "source:{}".format(ua_chroot) in out.decode("utf-8"):
```

Comment:
Instead of searching for the chroot, we could instead run this:
`schroot -i -c source:ua-xenial-amd64`

Reply:
oh sweet, I'll change it to use that

Reply:
@lucasmoura updated!